src/metakb/cli.py

```diff
@@ -7,401 +7,361 @@
+    ),
+)
+@click.option(
+    "--db_password",
+    default="password",
+    help=(
+        "Password to provide to application Neo4j database. Can also be provided via environment variable METAKB_DB_PASSWORD, which takes priority."
+    ),
+)
+@click.option(
+    "--force_load_normalizers_db",
+    "-f",
+    is_flag=True,
+    default=False,
+    help=("Load all normalizers data into DynamoDB database."),
+)
+@click.option(
+    "--normalizers_db_url",
+    help=(
+        "URL endpoint of normalizers DynamoDB database. Set to `http://localhost:8000` by default."
+    ),
+)
+@click.option(
+    "--load_latest_cdms",
+    "-l",
+    is_flag=True,
+    default=False,
+    help=(
+        "Clear MetaKB Neo4j database and load most recent available source CDM files. Does not run harvest and transform methods to generate new CDM files. Exclusive with --load_target_cdm and --load_latest_s3_cdms."
+    ),
+)
+@click.option(
+    "--load_target_cdm",
+    "-t",
+    type=click.Path(exists=True, dir_okay=False, readable=True, path_type=Path),
+    required=False,
+    help=(
+        "Load transformed CDM file at specified path. Exclusive with --load_latest_cdms and --load_latest_s3_cdms."
+    ),
+)
+@click.option(
+    "--load_latest_s3_cdms",
+    "-s",
+    is_flag=True,
+    default=False,
+    required=False,
+    help=(
+        "Clear MetaKB database, retrieve most recent data available from public VICC S3 bucket, and load the database with retrieved data. Exclusive with --load_latest_cdms and load_target_cdm."
+    ),
+)
+@click.option(
+    "--update_cached",
+    "-u",
+    is_flag=True,
+    default=False,
+    required=False,
+    help=(
+        "`True` if civicpy cache should be updated. Note this will take several minutes. `False` if local cache should be used"
+    ),
+)
+async def update_metakb_db(
+    db_url: str,
+    db_username: str,
+    db_password: str,
+    force_load_normalizers_db: bool,
+    normalizers_db_url: str,
+    load_latest_cdms: bool,
+    load_target_cdm: Optional[Path],
+    load_latest_s3_cdms: bool,
+    update_cached: bool,
+) -> None:
+    """Execute data harvest and transformation from resources and upload to graph
+    datastore.
+
+    :param db_url: URL endpoint for the application Neo4j database. Can also be provided
+        via environment variable ``METAKB_DB_URL``, which takes priority.
+    :param db_username: Username to provide to application Neo4j database. Can also be
+        provided via environment variable ``METAKB_DB_USERNAME``, which takes priority.
+    :param db_password: Password to provide to application Neo4j database. Can also be
+        provided via environment variable ``METAKB_DB_PASSWORD``, which takes priority.
+    :param force_load_normalizers_db: Load all normalizers data into DynamoDB database.
+    :param normalizers_db_url: URL endpoint of normalizers DynamoDB database. Set to
+        ``http://localhost:8000`` by default.
```

Comment:
This is still technically correct but I'd say we should either strike the "set to x by default" sentence or add a little context

```suggestion
    :param normalizers_db_url: URL endpoint of normalizers DynamoDB database. If not given,
        defaults to ``http://localhost:8000`` per the configuration rules of the individual 
        normalizers.
```